### ISSUE TITLE
Audit & Create Issues for Accessibility Problems

### ISSUE THREAD

**Issue by vkoves** [chihacknight/decarbonize-my-state#13]
We should use tools like [Axe DevTools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd) and manual QA to check for accessibility issues and make separate issues for them. In particular Viktor noticed off-hand:

- A few basic HTML issues (like missing the `lang` attribute on the `<html>` tag)
- Color contrast issues on the state emissions graph tooltip
- No keyboard behavior to let keyboard-only users see tooltips and see clear focus indicators when navigating through the state map

Here's a screenshot from Axe DevTools showing a few of these:

![Screenshot from 2022-02-16 12-26-11](https://user-images.githubusercontent.com/3187531/154331421-d54d6ecb-8ae1-4e1e-8160-47f37e43fa2c.png)

**Comment by vkoves** [comment]
I'm going to wait to tackle this until issue #19 and issue #27 are closed out, since we might get rid of some inaccessible elements

**Comment by vkoves** [comment]
I've done some testing with a screen reader and the only last issue is the graphs are not accessible in any way, but I don't think there's a clean fix for that, so I am closing out this issue and we can open a ticket for that specifically
